src/notes/StructuredFieldPlugin.jsx

```diff
@@ -612,11 +612,13 @@ function StructuredFieldPlugin(opts) {
                 for (const key of keyForShortcut) {
                     // Add two keys: The key of the node (an Inline) and its children (Text nodes)
                     const shortcutNode = state.document.getDescendant(key);
-                    for (const childNode of shortcutNode.nodes) {
-                        const childKey = childNode.key;
-                        allNodeKeys.push(childKey);
+                    if (shortcutNode) {
```

Comment:
[JA1] We should add this null check in the `else` case also on line 626.

Reply:
done!